### DISCUSSION TITLE
Multiple note buttons?

### DISCUSSION THREAD

**Discussion by KingMichaelCoy** [tttapa/Control-Surface#813] [General]
Hi

I am using the send midi notes code for my midi controller which is assigned to pin 22 of my Adafruit huzzah32. How do I get more than this one button to work. I have other pins assigned to the other five buttons but when I try to assign to the other corresponding buttons it does not trigger the other buttons. How do I make multiple buttons trigger midi notes and not just what's assigned to pin 22? Do I need a multiplexer? I do have one but it seems it should work without using this.

**Comment by tttapa** [comment]
Please create one thread per issue, and post your code as requested.

→ https://github.com/tttapa/Control-Surface/discussions/803
